SQL/New_patches/2024-12-18-Login_statistics.sql

```diff
@@ -0,0 +1,16 @@
+DROP TABLE IF EXISTS Login_Summary_Statistics;
```

Comment:
Don't drop tables in release patches. If something needs to be dropped (doesn't seem to be since it's created below), it should go in a cleanup patch.